### PR TITLE
ref: Node DB auto-instrumentation docs

### PR DESCRIPTION
**Pull request by kamilogorek**
Implementation: https://github.com/getsentry/sentry-javascript/pull/3483

### CODE REVIEW

**Review by HazAT — APPROVED** [review]
🥇  Let's merge this once we released the SDK

File: src/platforms/node/common/performance/database.mdx [hunk 1]
```diff
@@ -2,56 +2,27 @@
 title: Database integrations
 ---
 
-Node.js has a few integrations that support tracking database queries as Spans. For now you only need to use the latest version of our SDK and add the specific integration to `Sentry.init`:
+Node.js has a few integrations that support tracking database queries as Spans. Starting version `6.4.0`, when you are using one of the supported database drivers or ORMs, the integration is installed automatically for you, there is nothing you need to change.
```
Comment (lobsterkatie) [inline src/platforms/node/common/performance/database.mdx:5]:
```suggestion
Node.js has a few integrations that support tracking database queries as spans. Starting version `6.4.0`, `@sentry/tracing` will auto-detect supported database drivers or ORMs being used in your project, and automatically enable the relevant integrations with default options - no additional code necessary.
```

File: src/platforms/node/common/performance/database.mdx [hunk 2]
```diff
@@ -2,56 +2,27 @@
...
-    // ....
-    new Tracing.Integrations.Mongo(), // Add this integration
-  ],
-  // ....
-});
-```
+### Disabling automatic instrumentation
 
-### MySQL / MariaDB
+To remove an integration, please refer to our [integrations](/platforms/node/configuration/integrations/#removing-an-integration) docs.
```
Comment (lobsterkatie) [inline src/platforms/node/common/performance/database.mdx:11]:
```suggestion
To learn how to remove an automatically-enabled integration, please refer to our [integrations](/platforms/node/configuration/integrations/#removing-an-integration) docs.
```

I actually think we might to well to have an example here, even with the link.

**Review by lobsterkatie — CHANGES_REQUESTED** [review]
Can we also document a) how to configure an auto-enabled integration, and b) what options there are to configure for each?

File: src/platforms/node/common/performance/database.mdx [hunk 3]
```diff
@@ -2,56 +2,27 @@
 title: Database integrations
 ---
 
-Node.js has a few integrations that support tracking database queries as Spans. For now you only need to use the latest version of our SDK and add the specific integration to `Sentry.init`:
+Node.js has a few integrations that support tracking database queries as spans. Starting version `6.4.0`, `@sentry/tracing` will auto-detect supported database drivers or ORMs being used in your project, and automatically enable the relevant integrations with default options - no additional code necessary.
```
Comment (PeloWriter) [inline src/platforms/node/common/performance/database.mdx:5]:
```suggestion
Node.js integrations support tracking database queries as spans. Starting in version `6.4.0`, `@sentry/tracing` will auto-detect supported database drivers or ORMs being used in your project, and automatically enable the relevant integrations with default options - without needing additional code.
```

File: src/platforms/node/common/performance/database.mdx [hunk 4]
```diff
@@ -2,56 +2,27 @@
...
 
-### MongoDB
+Supported packages and their integration name are, respectively: `pg` (Postgres), `mongodb` (Mongo), `mongoose` (Mongo) and `mysql` (Mysql).
```
Comment (PeloWriter) [inline src/platforms/node/common/performance/database.mdx:7]:
```suggestion
Supported packages and their integration name: 
- `pg` (Postgres)
- `mongodb` (Mongo)
- `mongoose` (Mongo)
- `mysql` (MySQL)
```

File: src/platforms/node/common/performance/database.mdx [hunk 5]
```diff
@@ -2,56 +2,27 @@
...
-Sentry.init({
-  // ....
-  integrations: [
-    // ....
-    new Tracing.Integrations.Mysql(), // Add this integration
-  ],
-  // ....
-});
-```
+### Adding integrations manually
```
Comment (PeloWriter) [inline src/platforms/node/common/performance/database.mdx:13]:
```suggestion
### Manually Adding Integrations
```

**Review by PeloWriter — APPROVED** [review]
Minor edits to smooth.

**Review by ahmedetefy — APPROVED** [review]

**Pull request merged**